package.json

```diff
@@ -1,25 +1,39 @@
 {
   "name": "module-2",
   "version": "1.0.0",
-  "description": "Training module 2 on Node.js Program",
+  "description": "Training module 3 on Node.js Program",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "tasK:2.1": ""
+    "start": "npm run db:up && npm run task:3.1",
```

Comment:
Very-very bad idea to do `db:up` with service start - it can change db schema unexpectedly or wipe all the data in the DB

Reply:
Super weird how these are showing. Maybe I didn't push the right update. This looks like this on my side -- I've noticed this before with our PRs.

<img width="492" alt="Screen Shot 2020-02-09 at 10 37 28 AM" src="https://user-images.githubusercontent.com/16273131/74107761-2f25c180-4b28-11ea-8a5d-ae56706bead2.png">